### PR TITLE
created new file for springfield.txt

### PR DESCRIPTION
**Pull request by ananthuluva**

### CODE REVIEW

File: springfield.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+park place
+route 22
```
Comment (rvgidda) [inline springfield.txt]: bedbathandbeyond

File: springfield.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+park place
```
Comment (rvgidda) [inline springfield.txt:1]: Hi Ken

**Comment by rakinepalli** [comment]
@ananthuluva Great job!

**Comment by kishorebolinidi** [comment]
Hola :accept:

**Comment by githubteacher** [comment]
Wait until you get a :+1: :ship: or a :shipit:

**Pull request merged**